shared/ui-components/src/page_components/unit_tables.tsx

```diff
@@ -0,0 +1,103 @@
+import * as React from "react"
+import { BasicTable } from "@dahlia/ui-components/src/tables/basic_table"
+
+const unitTypeLabel = unitType => {
+  const unitTypeLabels = {
+    studio: "Studio",
+    one_bdrm: "1 BR",
+    two_bdrm: "2 BR",
+    three_bdrm: "3 BR",
+    four_bdrm: "4 BR"
+  }
+  return unitTypeLabels[unitType]
+}
+
+const unitAreaRange = (units: any) => {
+  let min = units[0].sq_ft
+  let max = units[0].sq_ft
+  units.forEach(unit => {
+    if (unit.sq_ft < min) min = unit.sq_ft
+    if (unit.sq_ft > max) max = unit.sq_ft
+  })
+  let range = `${min} square feet`
+  if (min != max) range += ` - ${max} square feet`
+  return range
+}
+
+const ordinalize = num => {
+  const standardSuffix = "th"
+  const oneToThreeSuffixes = ["st", "nd", "rd"]
+
+  const numStr = num.toString()
+  const lastTwoDigits = parseInt(numStr.slice(-2), 10)
+  const lastDigit = parseInt(numStr.slice(-1), 10)
+
+  let suffix = ""
+  if (lastDigit >= 1 && lastDigit <= 3 && !(lastTwoDigits >= 11 && lastTwoDigits <= 13)) {
+    suffix = oneToThreeSuffixes[lastDigit - 1]
+  } else {
+    suffix = standardSuffix
+  }
+
+  return `${num}${suffix}`
+}
+
+const unitFloorRange = (units: any) => {
```

Comment:
maybe getUnitFloorRange or unitGetFloorRange? It took me a minute of wondering why we needed an object to represent the floor and area ranges